torch/optim/lr_scheduler.py

```diff
@@ -79,7 +79,22 @@ def _copy(_param):
 
 
 class LRScheduler:
-    r"""Adjusts the learning rate during optimization."""
+    r"""Learning Rate Scheduler Base Class.
+
+    This is the base class for all learning rate schedulers in PyTorch. It provides
+    common functionality for adjusting the learning rate of an optimizer during training.
+
+    .. note::
+        You cannot use LRScheduler directly, see one of the concrete implementations and tutorials
+        `in here <https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate>`_.
+
+
+    Args:
+        optimizer (Optimizer): Wrapped optimizer. (Find more about
+            `optimizer <https://pytorch.org/docs/stable/optim.html#base-class>`_) .
+        last_epoch (int, optional): The index of the last training epoch to resume.
+            Default: -1, starts from the optimizer lr.
```

Comment:
```suggestion
            Default: -1, starts from the optimizer lr.
```

The starts from the optimizer lr is confusing still, see https://github.com/pytorch/pytorch/issues/120735#issuecomment-1974862083